test/karma/behaviorSpec.js

```diff
@@ -189,26 +158,57 @@ describe('A Torso Behavior', function() {
     } catch (e) {}
   });
 
-  it('options defined in the view are passed to the behavior in its "initialize"', function() {
-    var BehaviorWithInitialize = TorsoBehavior.extend({
-      initialize: function() {
-        this.initializeArguments = arguments;
-      }
-    });
-
+  it('behavior options defined in the view are passed to the behavior in its "initialize"', function() {
     var ViewWithBehaviorWithInitialize = TorsoView.extend({
       behaviors: {
-        behaviorWithInitialize: {
-          behavior: BehaviorWithInitialize,
+        behaviorRecordingInitializeArguments: {
+          behavior: BehaviorRecordingInitializeArguments,
           propertyKey: 'propertyValue'
         }
       }
     });
     var viewWithBehaviorWithInitialize = new ViewWithBehaviorWithInitialize();
 
-    var behaviorWithInitialize = viewWithBehaviorWithInitialize.getBehavior('behaviorWithInitialize');
-    expect(behaviorWithInitialize.initializeArguments).toBeDefined();
-    expect(_.isMatch(behaviorWithInitialize.initializeArguments[0])).toBe(true);
+    var behaviorRecordingInitializeArguments = viewWithBehaviorWithInitialize.getBehavior('behaviorRecordingInitializeArguments');
+    expect(behaviorRecordingInitializeArguments.initializeArguments).toBeDefined();
+    var recordedBehaviorOptions = behaviorRecordingInitializeArguments.initializeArguments[0];
+    expect(_.isMatch(recordedBehaviorOptions, {propertyKey: 'propertyValue'})).toBe(true);
+  });
+
+  it('view options are passed to behavior initialize', function() {
```

Comment:
While porting over an existing mixin to a Behavior, I realized Behaviors need access to a View's initialization options. Right now, I'm merging the options object passed into a View's initialization with the Behavior's options, but I don't like this because I think overriding properties is unintuitive. I'd prefer to either save the view's initialization options on the view, or pass in the options as a separate object. Thoughts?